llvm/include/llvm/MC/MCAsmInfo.h

```diff
@@ -471,6 +471,12 @@ class LLVM_ABI MCAsmInfo {
     return nullptr;
```

Comment:
We should add a bool parameter to `getNonexecutableStackSection` and rename it to `getStackSection`.